.github/workflows/ci.yml

```diff
@@ -0,0 +1,17 @@
+name: CI
+on:
+  push:
```

Comment:
I think adding `pull_request` would also be useful.
```suggestion
  push:
  pull_request:
```

Reply:
Done @straight-shoota. Thanks.